netbox/dcim/tables/columns.py

```diff
@@ -0,0 +1,21 @@
+import django_tables2 as tables
+
+__all__ = (
+    'SpeedColumn',
+)
+
+
+class SpeedColumn(tables.TemplateColumn):
+    """
+    Humanize the speed in the column view
+    """
+    template_code = """
+        {% load helpers %}
+        {{ value|humanize_speed|placeholder }}
```

Comment:
Using our own `netbox.tables.columns.TemplateColumn` will handle the placeholder for you, simplifying the column definition.

Reply:
Ah sorry, I mixed up `TemplateColumns`. Switching to `netbox.tables.columns.TemplateColumn` does indeed handle everything and makes this change much simpler. Thank you.